CMakeLists.txt

```diff
@@ -56,19 +56,3 @@ else()
 	add_subdirectory(3rdparty/hidapi/libusb)
 	#list(APPEND LIBS hidapi-libusb)
 endif()
-
-# Linux installation
```

Comment:
Why was this moved?
If it's for technical reasons than that's fine, but if it was purely management, I'd like to leave it in the same spot it was before. `rpcs3/CMakeLists.txt` is already crowded enough with all of the random build shenanigans, I'd like to use this CMake to host the installation stuff.

It's also standard practice as far as I'm concerned, both Dolphin https://github.com/dolphin-emu/dolphin/blob/master/CMakeLists.txt and Citra https://github.com/citra-emu/citra/blob/master/CMakeLists.txt have it this way.